verifiers/internal/gcb/provenance.go

```diff
@@ -448,16 +463,18 @@ func (p *Provenance) verifySignatures(prov *provenance) error {
 				errs = append(errs, err)
 				continue
 			}
+		} else {
+			continue
```

Comment:
is this else continue necessary?

Reply:
If we reach this 'else', it will mean this signature did not match the global PAE key, or the regional key regex. If we don't continue here, the 'Verification succeeded' code below will be executed

Reply:
AH I mixed up which lines were in that scope, thanks!